src/plonk/prover.rs

```diff
@@ -212,15 +251,30 @@ pub fn create_proof<
                 }
             }
 
+            let unusable_rows_start = params.n as usize - (meta.blinding_factors() + 1);
```

Comment:
This still works for me when `unusable_rows_start = params.n as usize - meta.blinding_factors()`. Is that expected?